### DISCUSSION TITLE
Use blade component

### DISCUSSION THREAD

**Discussion by AleJavierHY** [rawilk/laravel-breadcrumbs#7] [General]
Hi, I have been testing the package and everything works fine until I try to use the blade component to render the breadcrumbs.
I get this error: Unable to locate a class or view for component [breadcrumbs].

This is how I call the blade component in the view:

```
<x-app-layout>
    <x-slot name="header">
        <h2 class="font-semibold text-xl text-gray-800 leading-tight">
            {{ __('Dashboard') }}
        </h2>
    </x-slot>

    <x-breadcrumbs breadcrumbs="dashboard" />
</x-app-layout>
```
I am using Jetstream

**Comment by rawilk** [comment]
I'm not really sure why it wouldn't be working with Jetstream since I'm registering the blade component in my service provider. I don't use Jetstream myself, so I can't really say for sure.

Does it work if you try the namespaced name of it?

```html
<x-breadcrumbs::components.breadcrumbs breadcrumbs="dashboard" />
```

**Reply by AleJavierHY** [reply]
No, I tried both ways even with a Laravel installation without Jetstream and nothing, it didn't work. It keeps showing me the same error message. it only works for me with `{!! Breadcrumbs::render('dashboard') !!}`

**Reply by rawilk** [reply]
Interesting. The only other thing I can think of is running `php artisan view:clear`.

I'm not sure which Laravel version you're using, but I know it works with a Laravel 9 application, since I'm using it in the docs:
https://github.com/rawilk/randallwilk.dev/blob/main/resources/views/front/pages/docs/partials/header.blade.php#L3
